packaging/windows/clisdk/bundle.wxs

```diff
@@ -32,13 +34,13 @@
     <util:DirectorySearch Path="[PreviousInstallFolder]" Variable="DOTNETHOME" After="PreviousInstallFolderSearch" Condition="PreviousInstallFolder" />
 
     <Chain DisableSystemRestore="yes" ParallelCache="yes">
-      <MsiPackage SourceFile="$(var.CLISDKMsiSourcePath)">
+      <MsiPackage SourceFile="$(var.CLISDKMsiSourcePath)" Visible="yes">
```

Comment:
I reverted this change for now. The bundle is going to be in the ARP, not the MSIs.